agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/OpenTelemetryConfigurer.java

```diff
@@ -208,7 +211,40 @@ private static SpanExporter createSpanExporter(
       Configuration configuration,
       boolean captureHttpServer4xxAsError) {
 
-    SpanExporter spanExporter = new Exporter(telemetryClient, captureHttpServer4xxAsError);
+    AzureMonitorTraceExporter azureMonitorTraceExporter =
+        new AzureMonitorTraceExporter(
+            captureHttpServer4xxAsError,
+            telemetryClient::populateDefaults,
+            (event, instrumentationName) -> {
+              boolean lettuce51 = instrumentationName.equals("io.opentelemetry.lettuce-5.1");
```

Comment:
filtering is also better to be done inside the exporter, whichever class that needs to create such a new exporter, will need to pass in the same filter function?